pandas/tests/test_import.py

```diff
@@ -0,0 +1,14 @@
+def test_import_error():
```

Comment:
Think this test can go in `tests/test_common.py`

Reply:
I tried adding the test in the file `tests/test_common.py`, but if I run pytest (with pandas 0.23.4) on that file this test succeeds instead of fails. This because pandas will already be imported before warnings:

```python
import pandas as pd  # Defined at the beginning

def test_import_error():
    import warnings
    warnings.filterwarnings('error')
   import pandas as pd
```

Is there a way to completely unload the earlier import of pandas? I tried adding del('pandas') within the test, but that did not help.